src/analyze/analyze.c

```diff
@@ -2366,6 +2387,21 @@ static int parse_argv(int argc, char *argv[]) {
 
                         break;
 
+                case ARG_ALIAS:
+                        if (!streq(strstrip(optarg), "")) {
+                                _cleanup_free_ char *tempdir = NULL;
+
+                                r = mkdtemp_malloc("/tmp/systemd-analyze-bootid-XXXXXX", &tempdir);
```

Comment:
I meant the current bootid rather than a fixed string - have a look at what this does as an example:

https://github.com/systemd/systemd/blob/main/src/core/namespace.c#L2512